fvupdater.h

```diff
@@ -44,9 +48,16 @@ public slots:
     bool CheckForUpdatesSilent();
     bool CheckForUpdatesNotSilent();
 
+    // Update window button slots
+    void InstallUpdate();
+    void SkipUpdate();
+    void RemindMeLater();
+
 signals:
     // This signal will inform, whether network is accessible or not
     void updatesDownloaded(bool success);
+    void proposedUpdateChanged(FvAvailableUpdate* proposedUpdate);
```

Comment:
Add another simple signals:
 - proposedVersionChanged(QString version)
 - proposedReleaseNotesChanged(QString content)
 - proposedReleaseNotesLinkChanged(QURL link)

It is better to use basic Q-classes, than specific `FvAvailableUpdate` and their methods.